src/output-json-bittorrent-dht.c

```diff
@@ -0,0 +1,177 @@
+/* Copyright (C) 2020 Open Information Security Foundation
```

Comment:
Should there also be changes to output-json-alert.c ?

Reply:
Sure, I'll add changes for bittorrent-dht in output-json-alert :)